backend/molgenis-emx2-sql/src/main/java/org/molgenis/emx2/sql/SqlTable.java

```diff
@@ -354,7 +356,7 @@ private static void executeBatch(
     subclassRows.get(subclassName).clear();
   }
 
-  private static int insertBatch(
+  private int insertBatch(
```

Comment:
why these refactorings? I think the change should be kept much more local

Reply:
code to very tightly coupled , that makes it had to test and maintain